src/com/unbounce/treajure/concurrent/mvar.clj

```diff
@@ -0,0 +1,81 @@
+(ns com.unbounce.treajure.concurrent.mvar
+  "Haskell's MVar abstraction in Clojure.
+
+  Check: http://chimera.labs.oreilly.com/books/1230000000929/ch07.html for more
+  info around MVars."
```

Comment:
Put them on the project wiki => https://github.com/unbounce/treajure/wiki/Intro-to-MVar